docker/entrypoint.sh

```diff
@@ -1,29 +1,63 @@
 #!/bin/sh
 
 usage() {
-	echo "$0 --dns <IP> --hostname <NAME> --searchpath <SEARCHPATH> --app.foo bar --probe.foo bar"
+	echo "$0 --app.foo bar --probe.foo bar"
 	exit 1
 }
 
 # This script exists to modify the network settings in the scope containers
 # as docker doesn't allow it when started with --net=host
 
+WEAVE_CONTAINER_NAME=weave
+DOCKER_BRIDGE=docker0
+HOSTNAME=scope
+DOMAIN=weave.local
```

Comment:
Yeah that won't work.  That was broken before and this change has made it no worse - I'll file a bug for it.